account-kit/logging/src/types.ts

```diff
@@ -0,0 +1,11 @@
+export interface EventLogger {
+  trackEvent<
+    Name extends string = string,
+    Data extends Record<string, any> | undefined =
```

Comment:
should this be `Record<string, any>` or should we actually restrict it specific types here? Do we run the risk of logging things that we shouldn't if this is any?

Reply:
I was actually thinking that in specific packages, you might type this more strongly. this is related to the static logger question below, see my answer there

Reply:
but going off that... then this should actually move up higher and be generic on the `createLogger` function. maybe some type like `EventsSchema` and `Data extends EventsSchema[number]`...

hold up, gonna update this

Reply:
updated